bootstrap-salt.sh

```diff
@@ -4073,15 +4073,20 @@ __configure_freebsd_pkg_details() {
     mkdir -p /etc/pkg/
 
     ## Use new JSON-like format for pkg repo configs
-    conf_file=/usr/local/etc/pkg/repos/freebsd.conf
-    {
-        echo "FreeBSD:{"
-        echo "    url: \"${PKGCONFURL}\","
-        echo "    mirror_type: \"SRV\","
-        echo "    enabled: true"
-        echo "}"
-    } > $conf_file
-    copyfile $conf_file /etc/pkg/FreeBSD.conf
+    ## check if /etc/pkg/FreeBSD.conf is already in place 
+    if [ ! -f /etc/pkg/FreeBSD.conf ]; then 
+      conf_file=/usr/local/etc/pkg/repos/freebsd.conf
```

Comment:
@serge-p, would it be better to name this file and repo `saltstack.conf` and `SaltStack`?  Also, what happens when `/etc/pkg/FreeBSD.conf` does exist?